config/locales/en.yml

```diff
@@ -811,6 +811,8 @@ en:
       remember_me: "Remember me"
       reset_by_clicking_here: "Forgot your password? <a href='%{url1}'>Reset it here</a>"
   home:
+    read_more: "Read more"
+    more_projects: "More projects"
```

Comment:
Hi @KarishmaVanwari, there should be a home key before the read_more key. You can take a look at the issue again to see what I’m talking about.

Also the rest of the files added to this PR are not a part of the changes required. To remove them, all you have to do is undo the changes you made to them and then do a push again. Thank you!

Reply:
Hey @TildaDares the home key was aleready present as you can see on line 813.

Reply:
And those chnages to other files haven't ben made by me. It directly appeared maybe as a result of procedure to setup the project. So how do I revert it back now? @TildaDares

Reply:
Hi @KarishmaVanwari, there should be two `home` keys
<img width="427" alt="Screenshot 2021-11-25 at 08 11 05" src="https://user-images.githubusercontent.com/63427719/143396201-677ea6d5-0d52-41af-ace4-da830e413213.png">
s

Reply:
Let me know if this SO answer works for you https://stackoverflow.com/a/15321456/11953728

Reply:
> 

Ohh yes, I'm sorry

Reply:
> 

yes this answer worked, thank you @TildaDares

Reply:
@KarishmaVanwari The changes you made to this file are perfect but the other files have not been removed.